doc/api.rst

```diff
@@ -38,16 +38,16 @@ MNE BIDS
    get_anonymization_daysback
 
 
-Utils
```

Comment:
everything in `utils` is now in `path`? Or there is no public function in `utils`?

Reply:
Only `get_anonymization_dateback`, which I added back. However, I saw it under the main namespace, so forgot to add it in.